app/cmd/backup.go

```diff
@@ -144,6 +186,15 @@ func checkBackupStatus(c *cli.Context) error {
 		return fmt.Errorf("replica address is empty")
 	}
 
+	knownReplicas, err := getReplicaModeMap(c)
+	if err != nil {
+		return err
+	}
+	if mode, known := knownReplicas[replicaAddress]; !known || mode != types.RW {
```

Comment:
Need to update this part as well. Also, use `mode :=` instead of `mode, _ :=` since we normally omit the existence check if it's not used for map.